store/tikv/gcworker/gc_worker.go

```diff
@@ -351,61 +351,20 @@ func (w *GCWorker) deleteRanges(ctx context.Context, safePoint uint64) error {
 	regions := 0
 	for _, r := range ranges {
 		startKey, rangeEndKey := r.Range()
-		for {
-			select {
-			case <-ctx.Done():
-				return errors.New("[gc worker] gc job canceled")
-			default:
-			}
 
-			loc, err := w.store.GetRegionCache().LocateKey(bo, startKey)
-			if err != nil {
-				return errors.Trace(err)
-			}
+		deleteRangeTask := tikv.NewDeleteRangeTask(ctx, w.store, bo, startKey, rangeEndKey)
```

Comment:
Is the bo is long enough when encounter a huge table like 10TB?

Reply:
This is the same as what the old code did. The value of `GcDeleteRangeMaxBackoff` is 100000